ipatests/azure/scripts/setup_containers.py

```diff
@@ -195,7 +195,7 @@ def setup_container_overrides(self):
         """
         Set services known to not work in containers to be ignored
         """
-        for service in ('chronyd', 'nis-domainname'):
+        for service in ('nis-domainname'):
```

Comment:
``for service in ('nis-domainname',):`` or better use a list here.

Reply:
oh, yes, thank you!